api/env.template

```diff
@@ -4,10 +4,8 @@ DJANGO_SETTINGS_MODULE=conf.settings
 DJANGO_SECRET_KEY=example_key
 DJANGO_DEBUG_ENABLED=True
 
-BASE_URL=http://localhost:50280/
+CANONICAL_DOMAIN=localhost:50280
 ENVIRONMENT=local
-# List of comma-separated hosts/domain names, e.g., 127.17.0.1,local.app
-ALLOWED_HOSTS=localhost,172.17.0.1,host.docker.internal
```

Comment:
Okay, so, the issue is that the example tests make the network call _inside_ the container, and therefore need to use the port the app runs on in the container, rather than the host port. So either the tests need to refer to host networking _or_ use the port inside the container.

But you are right that the docker host IP/name needs to be in alternative domains in `env.docker` :+1:

Reply:
Thanks!